src/resources/postcss/tickets-admin/attendees/_table.pcss

```diff
@@ -29,4 +32,90 @@
 		border-bottom: 1px solid #e0e0e0;
 		color: #50575e;
 	}
+
+	.tec-tickets__admin-attendees-attendance-container {
+		padding: 20px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-heading {
+		align-items: center;
+		display: flex;
+		margin-bottom: 8px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-heading-label {
+		font-size: 14px;
+		font-weight: 700;
+		margin-right: 8px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-heading-border {
+		border-bottom: 1px solid #DCDCDE;
+		flex: 1;
+		height: 1px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-heading-total {
+		margin-left: 8px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-heading-total-label {
+		font-size: 13px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-heading-total-amt {
+		font-size: 12px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-list {
+		margin: 8px 0;
+		padding: 0 15px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-complete {
+		align-items: center;
+		display: flex;
+		justify-content: space-between;
+		margin-bottom: 8px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-complete-label {
+		font-size: 13px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-complete-amount {
+		font-size: 12px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-cancelled {
+		align-items: center;
+		display: flex;
+		justify-content: space-between;
+		margin-bottom: 12px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-cancelled-label {
+		font-size: 13px;
+	}
+
+	.tec-tickets__admin-attendees-attendance-type-cancelled-amt {
```

Comment:
I saw in another comment you said you fixed `-amt` to `amount. Just wanted to call out the one here and the one below as well.